ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -257,38 +258,98 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  getRemainingTime(firstDate, secondDate) {
+    const differenceInTime = Math.abs(secondDate - firstDate);
+    let differenceInDays = Math.ceil(differenceInTime / (1000 * 60 * 60 * 24));
+
+    // If weekday of date1 is bigger than date2 then it is a new week
+    // and we have to subtract the weekend days
+    if (firstDate.getUTCDay() > secondDate.getUTCDay()) {
+      differenceInDays -= 2;
+    } else {
+      differenceInDays -= 2 * Math.trunc(differenceInDays / 7);
```

Comment:
This needs a comment explaining the logic behind it.

Reply:
it subtracts the number of weekend days from `differenceInDays`. First it calculates the number of weeks in `differenceInDays` (Math.trunc(differenceInDays / 7) and then calculates the number of weekend days in it (`2 *`).

Reply:
it subtracts the weekend days from `differenceInDays`. `Math.trunc(differenceInDays / 7)` is the number of weeks in `differenceInDays` and `2 *` is the number of days in a weekend.

Reply:
If someone was to revisit the code after a while it will not be straight forward to understand the logic. That's why you should move this calculation in a constant with a suitable name, perhaps including a comment if still unclear.
e.g. 
```
const weekendDaysCount = 2; // saturday and sunday are considered weekend days
const weeksCount = Math.trunc(differenceInDays / 7); // converts days to weeks and keeps the count of full weeks
const weekendDaysToSubstract = weekendDaysCount * weeksCount; // counts the total weekend days
const differenceInDays -= weekendDaysToSubstract; // substracts the total weekend days from the total days
```